shell/platform/darwin/ios/framework/Headers/FlutterDartRunner.h

```diff
@@ -0,0 +1,22 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_FLUTTERDARTRUNNER_H_
+#define FLUTTER_FLUTTERDARTRUNNER_H_
+
+#import <Foundation/Foundation.h>
+
+#include "FlutterDartProject.h"
+#include "FlutterMacros.h"
+
+FLUTTER_EXPORT
+@interface FlutterDartRunner : NSObject
```

Comment:
Nit: Is there perhaps a better name for this class? Something that signified that it allows one to run Dart code without a rendering concern. `FlutterHeadlessDartRunner`, `FlutterBackgroundDartRunner`, ...

cc @cbracken, WDYT?

Reply:
`FlutterHeadlessDartRunner` seems a little more explicit. Either way, it's worth documenting expected usage here.

Reply:
Renamed to FlutterHeadlessDartRunner